schutzbot/run_image_tests.sh

```diff
@@ -58,7 +58,7 @@ run_test_case () {
     AZURE_CREDS=${AZURE_CREDS-/dev/null}
     OPENSTACK_CREDS=${OPENSTACK_CREDS-/dev/null}
     VCENTER_CREDS=${VCENTER_CREDS-/dev/null}
-    TEST_CMD="env $(cat $AZURE_CREDS $OPENSTACK_CREDS $VCENTER_CREDS) CHANGE_ID=$CHANGE_ID BUILD_ID=$BUILD_ID DISTRO_CODE=$DISTRO_CODE $TEST_RUNNER -test.v ${IMAGE_TEST_CASES_PATH}/${TEST_CASE_FILENAME}"
+    TEST_CMD="env $(cat $AZURE_CREDS $OPENSTACK_CREDS $VCENTER_CREDS) CHANGE_ID=${CHANGE_ID-master} BUILD_ID=$BUILD_ID DISTRO_CODE=$DISTRO_CODE $TEST_RUNNER -test.v ${IMAGE_TEST_CASES_PATH}/${TEST_CASE_FILENAME}"
```

Comment:
I think we should fallback to `BRANCH_NAME` if `CHANGE_ID` is not available. We cannot assume that the branch is `master`.

Reply:
hmm, in that case I could use `BRANCH_NAME` instead of `CHANGE_ID` everywhere, it seems to be set to the PR number anyway: from CI `BRANCH_NAME=PR-984`

Reply:
Sounds good to me. :)